docs/src/administration/users.md

```diff
@@ -3,136 +3,182 @@ title: "User administration"
+<--->
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md) to delete existing users.
 
-For example, if you invite a user with **Project Admin** role to a project on Platform.sh, but you haven't invited them to the remote repository on GitHub / GitLab, they might be [unable to clone the project](/administration/web/_index.md#git) locally.
+`platform user:delete`
 
-In this example, using either `platform get` with the CLI:
+  ```bash
+  platform user:delete user1@example.com
+  ```
 
-```bash
-$ platform get <projectID>
+  After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command `platform redeploy`.
 
-```
+{{< /codetabs >}}
 
-or the `git clone` command visible from the "Git" dropdown in the management console
+Once you delete a user, they can no longer access the project anymore.
+After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment.
+You can redeploy by clicking **Redeploy** in the console.
 
-```bash
-$ git clone git@github.com:user/github-repo.git Project Name
-```
 
-both would error with
+### Change existing permissions for environment types
 
-```bash
-Failed to connect to the Git repository: git@github.com:user/github-repo.git
+To change the user permissions, follow these steps:
 
-Please make sure you have the correct access rights and the repository exists.
-```
+{{< codetabs >}}
+---
+title=In the console
+file=none
+highlight=false
+---
+1. Go to your console and select the project where you want to change the user permissions.
+2. Under **Settings**, click **Access**.
+3. Select a user and change the permissions.
+4. Click **Save**.
 
-despite their **Project Admin** access to the project.
+<--->
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md)to change user permissions.
 
-This enhaces consistency over your source code, as the Platform.sh project functions as a read-only mirror of your remote repository. Otherwise, changes pushed directly to the project would be overwritten or deleted when commits are pushed via the integration. Platform.sh considers your integrated remote repository to be the "source of truth" as soon as it has been configured, and this caveat ensures that all commits go through the integration.
+`platform user:role`
 
-The best course of action is to have your access updated on the integrated repository. If for some reason that is not a quick change, you can still clone through the project using the legacy pattern (which will set the *project* as its remote), but again, it is not recommended that you commit to the project once you have done so:
+  ```bash
+  platform user:role user1@example.com -r contributor
```

Comment:
This isn't the right command. For one, `user:role` doesn't exist anymore. And also, the role definition here is for without environment types.
Example to set Viewer for Production environments and Contributor for Development environments:

```bash
platform user:update user1@example.com -r production:v,development:c
```